media-video/mjpg-streamer/mjpg-streamer-0_pre20120621-r2.ebuild

```diff
@@ -13,37 +13,40 @@ LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-INPUT_PLUGINS="input_testpicture input_control input_file input_uvc"
-OUTPUT_PLUGINS="output_file output_udp output_http output_autofocus output_rtsp"
+INPUT_PLUGINS="inputtestpicture inputcontrol inputfile inputuvc"
+OUTPUT_PLUGINS="outputfile outputudp outputhttp outputautofocus outputrtsp"
```

Comment:
i suggest ext use flags approach, look at e.g. ffmpeg FFTOOLS flag.

Reply:
or perhaps there's no point in disabling plugins at all?

Reply:
We need grouping across input-* and output-* plugins 'coz theirs function is little different. We need some from output and some from input (ie. not only from input)

Reply:
you still can have MJPG_STREAMER_INPUTS and MJPG_STREAMER_OUTPUTS